<?xml version="1.0" encoding="UTF-8"?>
<launch>    
    <arg name="frame" default="40" />
    <arg name="data_dir" default="/home/tavu/workspace/slambench2/data" />
    <arg name="num" default="100" />
    <node pkg="smoothnet_3d" name="file_keypts" type="file_keypts_node.py" args="$(arg num)" output="screen">
        <param name="ply_file" value="$(arg data_dir)/ply/f_$(arg frame)_vertices.ply" />
        <param name="keypts_file" value="$(arg data_dir)/keypts/f_$(arg frame)_keypoints.txt" />
        
    </node>
</launch>

 
